brion/plugin/compartmentReportBinary.cpp

```diff
@@ -705,7 +705,8 @@ void CompartmentReportBinary::_parseGIDs()
     for (int32_t i = 0; i < _header.numCells; ++i)
     {
         auto gid = get<int32_t>(ptr, i * SIZE_CELL_INFO_LENGTH);
-        lunchbox::byteswap(gid);
+        if (_header.byteswap)
```

Comment:
ouch!

Reply:
yes... and that's why unit tests are not the holy grail. If we had to test all the possible combinations they would be trillions. On the other hand, this should have been spotted during the review.